.github/workflows/test-ui-gh_runner.yml

```diff
@@ -123,10 +159,17 @@ jobs:
           [[ $retCode -ne 0 ]] && exit 1
           popd
 
+      - name: eclwatch-ui-test-content-artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: ${{ inputs.asset-name }}-ui_test-content
+          path: /home/runner/HPCCSystems-regression/log/*.json
+          if-no-files-found: error
+
```

Comment:
Typically we only upload the files on error or cancellation?

Reply:
Right, changed